charts/dawarich/templates/_helpers.tpl

```diff
@@ -62,17 +62,14 @@ app.kubernetes.io/part-of: {{ .Chart.Name }}
   value: {{ include "dawarich.hosts" . | quote }}
 # photon
 - name: 'STORE_GEODATA'
-  value: {{ if .Values.photon.deploy }}'false'{{ else }}'true'{{ end }}
-{{- if not .Values.photon.deploy }}
+  value: {{ if .Values.photon.enabled }}'false'{{ else }}'true'{{ end }}
+{{- if .Values.photon.enabled }}
 - name: PHOTON_API_HOST
-  value: photon.komoot.io
-- name: PHOTON_API_USE_HTTPS
-  value: 'true'
-{{- else }}
-- name: PHOTON_API_HOST
-  value: {{ template "common.fullname" . }}-photon:2322
+  value: {{ template "common.fullname" . }}-photon-loadbalancer:8080
 - name: PHOTON_API_USE_HTTPS
   value: 'false'
+{{- else }}
+# Photon is disabled, app will use built-in geodata storage
```

Comment:
This else is redundant and the comment is incorrect. Without photon there is no geodata. This can be completely removed.